content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -134,11 +139,13 @@ attempts to resynchronize iptables rules with the kernel. If it is
 every time any Service or Endpoint changes. This works fine in very
 small clusters, but it results in a lot of redundant work when lots of
 things change in a small time period. For example, if you have a
-Service backed by a Deployment with 100 pods, and you delete the
+Service backed by a {{< glossary_tooltip term_id="deployment" text="Deployment" >}}
+with 100 pods, and you delete the
 Deployment, then with `minSyncPeriod: 0s`, kube-proxy would end up
 removing the Service's Endpoints from the iptables rules one by one,
 for a total of 100 updates. With a larger `minSyncPeriod`, multiple
-Pod deletion events would get aggregated together, so kube-proxy might
+Pod deletion {{< glossary_tooltip term_id="event" text="events" >}} would get aggregated
```

Comment:
```suggestion
Pod deletion events would get aggregated
```
we're not talking about the Event API here.

Reply:
Thanks @sftim for the info, will update the change